website/addons/mendeley/templates/mendeley_widget.mako

```diff
@@ -0,0 +1,14 @@
+<%inherit file="project/addon/widget.mako" />
+<script type="text/javascript">
+window.contextVars = $.extend(true, {}, window.contextVars, {
+    mendeley: {
+        folder_id: '${list_id | js_str}'
+    }
+});
+</script>
+<link rel="stylesheet" href="/static/addons/mendeley/citations_widget.css">
```

Comment:
You are already doing `require('./citations_widget.css')` in `widget-cfg.js`, so this should be removed.